uk_election_ids/data/id_requirements.json

```diff
@@ -65,6 +65,12 @@
     "local.(braintree|broxtowe|craven|derby|mid-sussex|north-kesteven|north-west-leicestershire|pendle|watford|woking).*.2019-05-02": {
       "default": "pilot-2019"
     },
+    "local.city-of-london.*": {
+      "default": null
+    },
+    "local.city-of-london-alder.*": {
+      "default": null
+    },
```

Comment:
My opening gambit here was to make this `local.city-of-london*` but that didn't match the `-alder` case.
Having looked at the regex soup in the matching rules, I decided to just double up here rather than try to modify it.